classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+    @students.classroom = self
+  end
```

Comment:
👏 Good effort in implementing the method! 👏 However, it seems that when attempting to use this method, an error of `NoMethodError` arises. Let's troubleshoot and fix this issue to ensure smooth functionality:

- [ ] **NoMethodError Resolution:** It appears there's a `NoMethodError` when using the method. Double-check your code and verify that the method is defined correctly, verify that `@students.classroom` is ok or put the proper variable name.

![image](https://github.com/MostafaShahat2021/OOP-school-library/assets/55840999/0815a56d-5ecf-4f66-a8ba-6eec7ecd5b16)

By addressing this error, your method will work seamlessly. Keep up the great work! 🛠️🚀

Reply:
Hey @shahadat3669, thanks for your suggestion, I'll try to fix that 😃